src/openssh/patch/0001-Put-style-as-line-number-to-ssh-session-environment-.patch

```diff
@@ -0,0 +1,2651 @@
+From e0f1ac19ec241a54418dbe8e5eaf4a7595b62287 Mon Sep 17 00:00:00 2001
+From: Jing Kan <jika@microsoft.com>
+Date: Wed, 21 Oct 2020 20:14:41 +0800
+Subject: [PATCH] Put style as line number to ssh session environment variable
+
+---
+ config.guess | 586 +++++++++++++++++++++++++++------------------------
```

Comment:
the patch contains lots of unrelated changes. please simplify.

Reply:
Yes... I will update it .